client/utils/gatsby/layout-selector.tsx

```diff
@@ -1,14 +1,26 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-
 import {
   CertificationLayout,
   DefaultLayout
 } from '../../src/components/layouts';
 import FourOhFourPage from '../../src/pages/404';
 import { isChallenge } from '../../src/utils/path-parsers';
 
-export default function layoutSelector({ element, props }) {
+type Location = {
+  pathname: string;
+};
```

Comment:
```suggestion
interface Location {
  pathname: string;
}
```
Nitpick, we prefer interfaces over type aliases: https://contribute.freecodecamp.org/#/codebase-best-practices?id=interfaces-and-types